matcher/service/openreview_interface.py

```diff
@@ -237,36 +263,19 @@ def set_alternates(self, alternates_by_forum):
 
     def _get_quota_arrays(self):
         '''get `minimum` and `maximum` reviewer load arrays, accounting for custom loads'''
-        minimums = [int(self.config_note.content['min_papers']) for r in self.reviewers]
-        maximums = [int(self.config_note.content['max_papers']) for r in self.reviewers]
-
-        all_reviewers = { r: r for r in self.reviewers }
-        custom_load_edges = []
-        edges = []
-        edges = self.client.get_grouped_edges(
-                invitation = self.config_note.content['custom_load_invitation'],
-                head = self.config_note.content['match_group'],
-                select='tail,label,weight')
-        if edges:
-            custom_load_edges = edges[0]['values']
-
-        for edge in custom_load_edges:
-            if edge['tail'] in all_reviewers:
-                try:
-                    custom_load = int(edge['weight'])
-                except ValueError:
-                    raise MatcherError('invalid custom load weight')
-
-                if custom_load < 0:
-                    custom_load = 0
-
-                index = self.reviewers.index(edge['tail'])
-                maximums[index] = custom_load
-
-                if custom_load < minimums[index]:
-                    minimums[index] = custom_load
-            else:
-                self.logger.warn('Reviewer {} not found in pool'.format(edge['tail']))
+        minimums = [int(self.config_note.content['min_papers']) for _ in self.reviewers]
```

Comment:
I don't understand what you are changing here.

Reply:
This change simply replaces the unused variable `r` with an `_`.